controllers/delayed.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+
+var redisModel = require('../models/redis'),
+    q = require('q');
+
+
+module.exports = function (app) {
+    var getDelayedModel = function(req, res){
+        var dfd = q.defer();
+        redisModel.getStatus("delayed").done(function(delayed){
```

Comment:
Yeah, it looks as it looks but this seems to be the standard way of doing things in this project. For example, compare with [this file](https://github.com/ShaneK/Matador/blob/master/controllers/active.js)